content/partials/app-preview-builds.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+
+Codemagic allows you to preview your unsigned `.app` artifact on an iOS simulator and interact with the simulator via a web browser. You can easily test the functionalities of your app as well as system notifications and flows that require location data on different simulator devices without requiring a physical device.
+
+{{<notebox>}}
+Note that this feature is available for **teams** on the Pay as you go billing plan by default and can be enabled for teams on the annual plan on request.
+{{</notebox>}}
+
+## Running preview builds
+
+You can preview any `.app` artifact built in Codemagic that targets the iOS simulator. For such artifacts, there is a **Quick Launch** button available next to the artifact name on the build overview page.
+
+Clicking **Quick Launch** displays a configuration popup for selecting the simulator device and runtime. Once you click **Start**, Codemagic starts a preview build during which the simulator is booted and the app is installed on the simulator. 
+
+When the simulator is ready for use, you'll see the URL to access the simulator printed in the logs under the "Running iOS simulator" step as well as a link to open the simulator on the left panel of the build overview page. The simulator session remains active for a maximum of **20 minutes** or until you cancel the build. 
```

Comment:
As the link on sidebar is far more user friendly I'd try to highlight it over the URL in logs, which isn't even formatted as a proper href, rendering it difficult to use. For starters I presume that even documenting the sidebar link/button first would be enough.

Reply:
As the sidebar link may also be hard to spot as it appears there only when the simulator becomes available, I think it's still a good idea to point out that the URL is also available in the logs.